services/wallet/collectibles/manager.go

```diff
@@ -1027,3 +1028,78 @@ func (o *Manager) SearchCollections(ctx context.Context, chainID walletCommon.Ch
 	}
 	return nil, ErrNoProvidersAvailableForChainID
 }
+
+func (o *Manager) FetchCollectionSocialsAsync(uniqueID thirdparty.CollectibleUniqueID) error {
+	hasValidSocials, err := o.collectionsDataDB.HasValidSocialsForID(uniqueID.ContractID)
+	if err != nil {
+		log.Debug("FetchCollectionSocialsAsync failed for", "chainID", uniqueID.ContractID.ChainID, "uniqueID", uniqueID, "err", err)
+		return err
+	}
+
+	if !hasValidSocials {
```

Comment:
Would probably be better to always send some response message for a request, even if we have the socials already, just skip fetching it from the providers